build-tools/src/main/kotlin/org/jetbrains/kotlin/MPPTools.kt

```diff
@@ -128,14 +129,19 @@ fun mergeReports(reports: List<File>): String {
     }
 }
 
-fun getCompileOnlyBenchmarksOpts(project: Project, defaultCompilerOpts: List<String>) =
-        (project.findProperty("nativeBuildType") as String?)?.let {
-            if (it.equals("RELEASE", true))
-                listOf("-opt")
-            else if (it.equals("DEBUG", true))
-                listOf("-g")
-            else listOf()
-        } ?: defaultCompilerOpts
+fun getCompileOnlyBenchmarksOpts(project: Project, defaultCompilerOpts: List<String>): List<String> {
+    val dist = project.file(project.findProperty("kotlin.native.home") ?: "dist")
```

Comment:
There is a project externsion property `kotlinNativeDist` that can be used here to reduce amount of same properties to be searched for.

Reply:
I can't use it, because 1) there is an old deprecated name, 2) I need guarantee that I don't use `konan.home`, because it's part of flow with multiplatform plugin and I should have same behaviour.